src/services/notificationServiceApi.js

```diff
@@ -143,6 +143,7 @@ export async function getJurorEmail(address) {
       return {
         email: jsonResponse?.email,
         error: null,
+        needsSignature: false,
```

Comment:
Do we need to return this? It seems weird that only this `ok` response would be returning `needsSignature`.

Reply:
nop, forgot to remove it after testing